src/els_code_navigation.erl

```diff
@@ -51,9 +51,8 @@ goto_definition(Uri, #{ kind := record_expr, id := Record }) ->
 goto_definition(_Uri, #{ kind := Kind, id := Include }
                ) when Kind =:= include;
                       Kind =:= include_lib ->
-  FileName = filename:basename(Include),
-  M = list_to_atom(FileName),
-  case els_utils:find_module(M) of
+  H = list_to_atom(filename:basename(Include, filename:extension(Include))),
```

Comment:
The "module" table is now gone, in favour of an indexed attribute on the "document" table. Now, header files are stored as "documents" as well and their id does not include the 'hrl' extension anymore, hence this change.

Reply:
Fixed.